package.sh

```diff
@@ -66,11 +66,56 @@ generate_rpm_package()
 	rpmbuild --define "_topdir ${STG_DIR}/${rpm_dir}${rpmbuild_dir}" -v -bb ${STG_DIR}/${rpm_dir}/tmp/aznfs.spec
 }
 
+generate_tarball_package() {
+    local arch=$1
+    local tar_pkg_dir
+    local compiler
+
+    if [ "$arch" == "amd64" ]; then
+        tar_pkg_dir=$tar_pkg_dir_amd64
+        compiler="gcc"
+    elif [ "$arch" == "arm64" ]; then
+        tar_pkg_dir=$tar_pkg_dir_arm64
+        compiler="aarch64-linux-gnu-gcc"
+    else
+        echo "Unsupported architecture: $arch"
+        exit 1
```

Comment:
nit, just to be consistent with other places, just fail this function instead of exiting.
so, just say

return 1